src/test/resources/doc/examplesExpected/sequence.xsd

```diff
@@ -28,13 +25,14 @@
     </xs:annotation>
     <xs:complexType>
       <xs:attribute ref="active" />
-      <xs:attribute name="elementRole" type="xs:string" fixed="roleEpsilon" use="prohibited" />
       <xs:attribute name="className" type="xs:string" fixed="org.frankframework.frankdoc.testtarget.examples.sequence.Opaque" use="prohibited" />
+      <xs:attribute name="elementRole" type="xs:string" fixed="roleEpsilon" use="prohibited" />
```

Comment:
Waarom is hier (en op andere plekken) de volgorde veranderd? 
Als je de volgorde dan toch aanpast, zet active dan onderaan.

Reply:
Dit kwam zo uit toen ik aan het refactoren was. Ik wilde zo veel mogelijk code hergebruiken. De volgorde van className en elementRole maakt niet uit voor de werking van de XSDs.

Reply:
Ik heb voor FrankElementen die als element worden uitgeschreven (geen type) de attributes "active" en "className" omgewisseld. Nu komt "className" eerst. De plaats van "roleName" is moeilijker te veranderen, omdat die in een andere functie wordt toegevoegd.